src/OpenTelemetry.Exporter.Geneva/README.md

```diff
@@ -1,6 +1,176 @@
-# Geneva Exporters for OpenTelemetry .NET
+# Geneva Exporter for OpenTelemetry .NET
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Exporter.Geneva.svg)](https://www.nuget.org/packages/OpenTelemetry.Exporter.Geneva)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Exporter.Geneva.svg)](https://www.nuget.org/packages/OpenTelemetry.Exporter.Geneva)
 
-TBD
+The Geneva Exporter exports telemetry to ETW or to a UDS on the local machine.
```

Comment:
Consider adding links and explanations for ETW and UDS (especially UDS because it's not an established term).

Reply:
I could yank the bottom references section and just make these acronyms links

Reply:
Its best to include a link to some doc, the 1st time we use the word "ETW/UDS"